translator/trace/README.md

```diff
@@ -0,0 +1,74 @@
+
+- `status.code` and `status.message`
+- `http.status_code` and `http.status_message`
+
+When converting from Jaeger to OC, 
+
+1. OC status should be set from `status.code` and `status.message` tags if `status.code` tag is found on the Jaeger span. Since OC already has a special status field, these tags (`status.code` and `status.message`) are redundant and should be dropped from resultant OC span.
+2. If the `status.code` tag is not present, status should be set from `http.status_code` and `http.status_message` if the `http.status_code` tag is found. HTTP status code should be mapped to the appropriate gRPC status code before using it in OC status. These tags should be preserved and added to the resultant OC span as attributes.
+3. If none of the tags are found, OC status should not be set.
+
+
+### Zipkin to OC
+
+In addition to the two sets of tags mentioned in the previous section, Zipkin spans can possibly contain a third set of tags to represent operation status resulting in the following sets of tags:
+
+- `census.status_code` and `census.status_description`
+- `status.code` and `status.message`
+- `http.status_code` and `http.status_message`
+
+When converting from Zipkin to OC,
+
+1. OC status should be set from `census.status_code` and `census.status_description` if `census.status_code` tag is found on the Zipkin span. These tags should be dropped from the resultant OC span.
+2. If the `census.status_code` tag is not found in step 1, OC status should be set from `status.code` and `status.message` tags if the `status.code` tag is present. The tags should be dropped from the resultant OC span.
+3. If no tags are found in step 1 and 2, OC status should be set from `http.status_code` and `http.status_message` if either `http.status_code` tag is found. These tags should be preserved and added to the resultant OC span as attributes.
+4. If none of the tags are found, OC status should not be set.
+
+
+Note that codes and messages from different sets of tags should not be mixed to form the status field. For example, OC status should not contain code from `http.status_code` but message from `status.message` and vice-versa. Both fields must be set from the same set of tags even if it means leaving one of the two fields empty.
+
+
+### OC to Jaeger
+
+When converting from OC to Jaeger, if the OC span has a status field set, then
+
+* `code` should be added as a `status.code` tag.
+* `message` should be added as a `status.message` tag.
+
+### OC to Zipkin
+
+When converting from OC to Zipkin, if the OC span has the status field set, then
+
+* `code` should be added as a `census.status_code` tag.
+* `message` should be added as a `census.status_description` tag.
+
+In addition to this, if the OC status field represents a non-OK status, then a tag with the key `error` and value `true` should be added to the Zipkin span if one is not already present. 
```

Comment:
please do not do this. zipkin has always regarded the "error" true an anti pattern. instrumentation have been known to set false nonsense value

the value of the "error" tag should be the best available message ex status code if a text variant is not available. empty message is ok and better than true.